playground/PostgresEndToEnd/PostgresEndToEnd.AppHost/Program.cs

```diff
@@ -21,6 +21,13 @@
 // External resources.
 var db10 = builder.AddPostgres("pg10").WithPgAdmin().PublishAsConnectionString().AddDatabase("db10");
 
+builder.Configuration["Parameters:pass"] = "p!ssw0rd1";
```

Comment:
Please use `p@ssw0rd1` as we already have a suppression setup for it.

Reply:
Its not url encoded.